Added labels

RonSherfey opened the pull request:
<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
v                               ✰  Thanks for creating a PR! ✰    
v    Before smashing the submit button please review the checkboxes.
v    If a checkbox is n/a - please still include it but + a little note why
☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->

Closes: #XXX

## Description

<!-- Add a description of the changes that this PR introduces and the files that
are the most critical to review.
-->

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [x] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [x] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [x] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [x] Reviewers assigned
- [x] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

ansermino commented on Makefile:
```diff
@@ -19,32 +19,37 @@ DOCKER_TAG = unstable
 DOCKER_IMAGE = cosmos/ethermint
 ETHERMINT_DAEMON_BINARY = emintd
 ETHERMINT_CLI_BINARY = emintcli
+GO_MOD=GO111MODULE=on
```
It should be possible to simplify this to `GO111MODULE=on`, with the additions below removed. Once its declared here it is within the scope of all the commands this file calls (eg. `go build`).

austinabell replied:
Tried this, doesn't work when GO111MODULE=on is not exported and code is run within the $GOPATH. I ran into this issue before and was very picky about being included before every go command which required the dependencies.

ansermino replied:
It seems to work for me if I just make this change:
```
GO111MODULE=on
GO_MOD=
```
Any chance you have `GO111MODULE` set on your machine already? Maybe in `.bashrc` or `.profile`?

austinabell replied:
Doesn't work to just do this when run in $GOPATH, although is sufficient for running code outside of $GOPATH.

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 850.26kb -> 546.77kb (35.69%)

/docs/guides/img/metamask_import.png -- 159.14kb -> 78.42kb (50.72%)
/docs/guides/img/metamask_network_settings.png -- 251.81kb -> 130.95kb (48%)
/docs/guides/img/remix_interact.png -- 152.80kb -> 113.48kb (25.73%)
/docs/guides/img/remix_deployed.png -- 130.73kb -> 99.81kb (23.65%)
/docs/guides/img/remix_deploy.png -- 126.90kb -> 98.20kb (22.62%)
/docs/basics/img/photon.png -- 18.56kb -> 15.65kb (15.68%)
/docs/.vuepress/public/logo.svg -- 5.24kb -> 5.20kb (0.73%)
/docs/.vuepress/public/logo-bw.svg -- 5.08kb -> 5.05kb (0.63%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

dependabot[bot] committed:
build(deps): bump actions/setup-node from v2.1.2 to v2.1.5

Bumps [actions/setup-node](https://github.com/actions/setup-node) from v2.1.2 to v2.1.5.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2.1.2...46071b5c7a2e0c34e49c3cb8a0e792e86e18d5ea)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump gaurav-nelson/github-action-markdown-link-check

Bumps [gaurav-nelson/github-action-markdown-link-check](https://github.com/gaurav-nelson/github-action-markdown-link-check) from 1.0.7 to 1.0.12.
- [Release notes](https://github.com/gaurav-nelson/github-action-markdown-link-check/releases)
- [Commits](https://github.com/gaurav-nelson/github-action-markdown-link-check/compare/1.0.7...0fe4911067fa322422f325b002d2038ba5602170)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump y18n from 3.2.1 to 3.2.2 in /tests-solidity

Bumps [y18n](https://github.com/yargs/y18n) from 3.2.1 to 3.2.2.
- [Release notes](https://github.com/yargs/y18n/releases)
- [Changelog](https://github.com/yargs/y18n/blob/master/CHANGELOG.md)
- [Commits](https://github.com/yargs/y18n/commits)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump pug-code-gen from 2.0.2 to 2.0.3 in /docs

Bumps [pug-code-gen](https://github.com/pugjs/pug) from 2.0.2 to 2.0.3.
- [Release notes](https://github.com/pugjs/pug/releases)
- [Commits](https://github.com/pugjs/pug/compare/pug-code-gen@2.0.2...pug@2.0.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump elliptic from 6.5.3 to 6.5.4 in /docs

Bumps [elliptic](https://github.com/indutny/elliptic) from 6.5.3 to 6.5.4.
- [Release notes](https://github.com/indutny/elliptic/releases)
- [Commits](https://github.com/indutny/elliptic/compare/v6.5.3...v6.5.4)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump ini from 1.3.5 to 1.3.8 in /tests-solidity

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.8.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.8)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump actions/cache from v2.1.2 to v2.1.5

Bumps [actions/cache](https://github.com/actions/cache) from v2.1.2 to v2.1.5.
- [Release notes](https://github.com/actions/cache/releases)
- [Commits](https://github.com/actions/cache/compare/v2.1.2...1a9e2138d905efd099035b49d8b7a3888c653ca8)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump github.com/tendermint/tendermint

Bumps [github.com/tendermint/tendermint](https://github.com/tendermint/tendermint) from 0.33.7 to 0.34.10.
- [Release notes](https://github.com/tendermint/tendermint/releases)
- [Changelog](https://github.com/tendermint/tendermint/blob/v0.34.10/CHANGELOG.md)
- [Commits](https://github.com/tendermint/tendermint/compare/v0.33.7...v0.34.10)

Signed-off-by: dependabot[bot] <support@github.com>

RonSherfey committed:
Merge pull request #5 from RonSherfey/dependabot/npm_and_yarn/tests-solidity/y18n-3.2.2

build(deps): bump y18n from 3.2.1 to 3.2.2 in /tests-solidity

RonSherfey committed:
Merge pull request #17 from RonSherfey/dependabot/go_modules/github.com/tendermint/tendermint-0.34.10

build(deps): bump github.com/tendermint/tendermint from 0.33.7 to 0.34.10

dependabot[bot] committed:
build(deps): bump ssri from 6.0.1 to 6.0.2 in /docs

Bumps [ssri](https://github.com/npm/ssri) from 6.0.1 to 6.0.2.
- [Release notes](https://github.com/npm/ssri/releases)
- [Changelog](https://github.com/npm/ssri/blob/v6.0.2/CHANGELOG.md)
- [Commits](https://github.com/npm/ssri/compare/v6.0.1...v6.0.2)

Signed-off-by: dependabot[bot] <support@github.com>

RonSherfey committed:
Merge pull request #18 from RonSherfey/dependabot/npm_and_yarn/docs/ssri-6.0.2

build(deps): bump ssri from 6.0.1 to 6.0.2 in /docs

RonSherfey committed:
Merge pull request #16 from RonSherfey/dependabot/github_actions/actions/cache-v2.1.5

build(deps): bump actions/cache from v2.1.2 to v2.1.5

dependabot[bot] committed:
build(deps): bump prismjs from 1.21.0 to 1.23.0 in /docs

Bumps [prismjs](https://github.com/PrismJS/prism) from 1.21.0 to 1.23.0.
- [Release notes](https://github.com/PrismJS/prism/releases)
- [Changelog](https://github.com/PrismJS/prism/blob/master/CHANGELOG.md)
- [Commits](https://github.com/PrismJS/prism/compare/v1.21.0...v1.23.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump y18n from 4.0.0 to 4.0.3 in /docs

Bumps [y18n](https://github.com/yargs/y18n) from 4.0.0 to 4.0.3.
- [Release notes](https://github.com/yargs/y18n/releases)
- [Changelog](https://github.com/yargs/y18n/blob/y18n-v4.0.3/CHANGELOG.md)
- [Commits](https://github.com/yargs/y18n/compare/v4.0.0...y18n-v4.0.3)

Signed-off-by: dependabot[bot] <support@github.com>

RonSherfey committed:
Merge pull request #19 from RonSherfey/dependabot/npm_and_yarn/docs/prismjs-1.23.0

build(deps): bump prismjs from 1.21.0 to 1.23.0 in /docs

dependabot[bot] committed:
build(deps): bump gopkg.in/yaml.v2 from 2.3.0 to 2.4.0

Bumps [gopkg.in/yaml.v2](https://github.com/go-yaml/yaml) from 2.3.0 to 2.4.0.
- [Release notes](https://github.com/go-yaml/yaml/releases)
- [Commits](https://github.com/go-yaml/yaml/compare/v2.3.0...v2.4.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump github.com/spf13/cobra from 1.1.0 to 1.1.3

Bumps [github.com/spf13/cobra](https://github.com/spf13/cobra) from 1.1.0 to 1.1.3.
- [Release notes](https://github.com/spf13/cobra/releases)
- [Changelog](https://github.com/spf13/cobra/blob/master/CHANGELOG.md)
- [Commits](https://github.com/spf13/cobra/compare/v1.1.0...v1.1.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump github.com/ethereum/go-ethereum from 1.9.21 to 1.10.2

Bumps [github.com/ethereum/go-ethereum](https://github.com/ethereum/go-ethereum) from 1.9.21 to 1.10.2.
- [Release notes](https://github.com/ethereum/go-ethereum/releases)
- [Commits](https://github.com/ethereum/go-ethereum/compare/v1.9.21...v1.10.2)

Signed-off-by: dependabot[bot] <support@github.com>

RonSherfey committed:
Merge pull request #14 from RonSherfey/dependabot/go_modules/gopkg.in/yaml.v2-2.4.0

build(deps): bump gopkg.in/yaml.v2 from 2.3.0 to 2.4.0

RonSherfey committed:
Merge pull request #13 from RonSherfey/dependabot/go_modules/github.com/ethereum/go-ethereum-1.10.2

build(deps): bump github.com/ethereum/go-ethereum from 1.9.21 to 1.10.2

RonSherfey committed:
Merge pull request #11 from RonSherfey/dependabot/go_modules/github.com/spf13/cobra-1.1.3

build(deps): bump github.com/spf13/cobra from 1.1.0 to 1.1.3

RonSherfey committed:
Merge pull request #9 from RonSherfey/dependabot/npm_and_yarn/tests-solidity/ini-1.3.8

build(deps): bump ini from 1.3.5 to 1.3.8 in /tests-solidity

RonSherfey committed:
Merge pull request #8 from RonSherfey/dependabot/npm_and_yarn/docs/elliptic-6.5.4

build(deps): bump elliptic from 6.5.3 to 6.5.4 in /docs

RonSherfey committed:
Merge pull request #7 from RonSherfey/dependabot/npm_and_yarn/docs/y18n-4.0.3

build(deps): bump y18n from 4.0.0 to 4.0.3 in /docs

dependabot[bot] committed:
build(deps): bump ini from 1.3.5 to 1.3.8 in /docs

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.8.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.8)

Signed-off-by: dependabot[bot] <support@github.com>

RonSherfey committed:
Merge pull request #20 from RonSherfey/dependabot/npm_and_yarn/docs/ini-1.3.8

build(deps): bump ini from 1.3.5 to 1.3.8 in /docs

RonSherfey committed:
Merge pull request #6 from RonSherfey/dependabot/npm_and_yarn/docs/pug-code-gen-2.0.3

build(deps): bump pug-code-gen from 2.0.2 to 2.0.3 in /docs

RonSherfey committed:
Merge pull request #3 from RonSherfey/dependabot/github_actions/gaurav-nelson/github-action-markdown-link-check-1.0.12

build(deps): bump gaurav-nelson/github-action-markdown-link-check from 1.0.7 to 1.0.12

RonSherfey committed:
Merge pull request #2 from RonSherfey/dependabot/github_actions/actions/setup-node-v2.1.5

build(deps): bump actions/setup-node from v2.1.2 to v2.1.5

RonSherfey committed:
Merge pull request #1 from RonSherfey/imgbot

[ImgBot] Optimize images

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ChainSafe/ethermint?pullRequest=1) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ChainSafe/ethermint?pullRequest=1) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: RonSherfey<br/>:x: ImgBotApp<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ChainSafe/ethermint?pullRequest=1) it.</sub>

RonSherfey reviewed (commented):
Requesting changes.

RonSherfey committed:
Create SECURITY.md

RonSherfey committed:
Create go.yml

RonSherfey reviewed (commented):
updates 1.0
